.deploy.yml

```diff
@@ -1,11 +1,14 @@
 trigger:
-- v3
-- feature/*
+  batch: true
```

Comment:
> batch: true [](start = 1, length = 12)

> If you set batch to true, when a build is running, the system waits until the build is completed, then queues another build of all changes that have not yet been built

is this the behavior we want when running PR validation?

Reply:
yep, to reduce # of builds